hygen/_templates/slice/new/prompt.js

```diff
@@ -0,0 +1,25 @@
+// see types of prompts:
+// https://github.com/enquirer/enquirer/tree/master/examples
+//
+const { promptArgs } = require('../../promptUtils');
+
+const prompt = async ({ prompter, args }) => {
+  let questions = [];
+
+  if (!args.name) {
+    questions.push({
+      type: 'input',
+      name: 'name',
+      message: 'Name:',
+    });
+  }
+
+  let { name } = await promptArgs({ prompter, args, questions });
+  name = name.replace('Slice', '').replace('slice', '');
```

Comment:
checkName here!

Reply:
No, because checkName also add the suffix, and in this case we don't want that. If I use `checkName` there I will be forced to remove the suffix later for name.

Reply:
Ok!